src/widgets/numeric-refinement-list/numeric-refinement-list.js

```diff
@@ -93,15 +94,18 @@ function numericRefinementList({
         templates
       });
 
-      this._toggleRefinement = facetValue => helper
-        .setState(refine(helper.state, attributeName, options, facetValue))
-        .search();
+      this._toggleRefinement = (facetValue) => {
```

Comment:
Conceptually, there are no difference.

I refactored it while debugging to more easily inspect the new state. In the end I decided to keep the change, but it does not change anything, it's just (maybe) more readable.